content/pt-br/docs/concepts/policy/resource-quotas.md

```diff
@@ -0,0 +1,649 @@
+- Detecte a demanda de um _namespace_, adicione nós e aumente a cota.
+
+Tais políticas podem ser implementadas usando `ResourceQuotas` como blocos de construção, por
+escrevendo um "controlador" que observa o uso da cota e ajusta os limites rígidos da cota de cada _namespace_ de acordo com outros sinais.
+
+Observe que a cota de recursos divide os recursos agregados do cluster, mas não cria restrições em torno dos nós: pods de vários _namespaces_ podem ser executados no mesmo nó.
+
+## Limite de consumo de classe de prioridade por padrão
+
+Pode ser desejado que os pods com uma prioridade particular, por exemplo. "cluster-services",
+deve ser permitido em um _namespace_, se, e somente se, existir um objeto de cota correspondente.
+
+Com este mecanismo, os operadores podem restringir o uso de certas classes de prioridade para um número limitado de _namespaces_ , e nem todos poderão consumir essas classes de prioridade por padrão.
+
+Para impor isso, a flag `kube-apiserver` `--admission-control-config-file` deve ser
+usada para passar o caminho para o seguinte arquivo de configuração:
+
+```yaml
+apiVersion: apiserver.config.k8s.io/v1
+kind: AdmissionConfiguration
+plugins:
+- name: "ResourceQuota"
+  configuration:
+    apiVersion: apiserver.config.k8s.io/v1
+    kind: ResourceQuotaConfiguration
+    limitedResources:
+    - resource: pods
+      matchScopes:
+      - scopeName: PriorityClass
+        operator: In
+        values: ["cluster-services"]
+```
+
+Em seguida, crie um objeto de cota de recurso no _namespace_ `kube-system`:
+
+{{< codenew file="policy/priority-class-resourcequota.yaml" >}}
+
+```shell
+kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
+```
+
+```none
+resourcequota/pods-cluster-services created
+```
+
+Nesse caso, a criação de um pod será permitida se:
+
+1. O `priorityClassName` do pod não foi especificado.
+1. O `priorityClassName` do pod é especificado com um valor diferente de `cluster-services`.
+1. O `priorityClassName` do pod está definido como `cluster-services`, ele deve ser criado no namespace `kube-system` e passou na verificação de cota de recursos.
+
+Uma solicitação de criação de pod é rejeitada se seu `priorityClassName` estiver definido como `cluster-services` e deve ser criado em um _namespace_ diferente de `kube-system`.
+
+## {{% heading "whatsnext" %}}
+
+- Veja [Documento de design de ResourceQuota](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_resource_quota.md) para mais informações.
+- Veja um [exemplo detalhado de como usar a cota de recursos](/docs/tasks/administer-cluster/quota-api-object/).
+- Leia [documento de design de suporte de cota para prioridade de classe](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-priority-resourcequota.md).
+- Veja [LimitedResources](https://github.com/kubernetes/kubernetes/pull/36765)
```

Comment:
"LimitedResources" should be translated to "Recursos Limitados"

Reply:
In the example `admission-control-config-file.yaml` in https://github.com/kubernetes/kubernetes/pull/36765 ,
there exists a field `limitedResources`.

And it seems that `LimitedResources` here indicates that field. 😊

Reply:
Thank you @jihoon-seo for your comment!

@brendavsantos no need to translate it!
Thanks